cql3/statements/index_prop_defs.cc

```diff
@@ -36,15 +33,6 @@ void cql3::statements::index_prop_defs::validate() {
                         db::index::secondary_index::custom_index_option_name));
     }
 
-    // Currently, Scylla does not support *any* class of custom index
-    // implementation. If in the future we do (e.g., SASI, or something
-    // new), we'll need to check for valid values here.
-    if (is_custom && custom_class) {
-        throw exceptions::invalid_request_exception(
-                format("Unsupported CUSTOM INDEX class {}. Note that currently, Scylla does not support SASI or any other CUSTOM INDEX class.",
-                        *custom_class));
-
-    }
```

Comment:
What will happen if you try to create a CUSTOM index, without specifying any custom class? I'm not sure it should be allowed.

Reply:
It is not allowed, it's checked in test_secondary_index_create_custom_index